client/src/components/DataFiles/DataFilesProjectMembers/DataFilesProjectMembers.module.scss

```diff
@@ -0,0 +1,87 @@
+.root {
+  display: block;
+}
+
+.access {
+  text-transform: capitalize;
+}
+
+.member-action {
+  display: flex;
+  align-items: center;
+  h6 {
+    margin-bottom: 0;
+    font-size: 10px;
```

Comment:
Think these font sizes should be 12px at least, maybe even 14. @tacc-wbomar what is the mockup to reality ratio again?